app/build.gradle

```diff
@@ -32,34 +32,41 @@ android {
         manifestPlaceholders.isRaptorEnabled = "false"
     }
 
+    def releaseTemplate = {
+        shrinkResources true
+        minifyEnabled true
+        proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        matchingFallbacks = ['release']
+    }
+
     buildTypes {
-        release {
-            shrinkResources true
-            minifyEnabled true
-            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
-        }
-        releaseRaptor {
-            initWith release
-            manifestPlaceholders.isRaptorEnabled = "true"
-            matchingFallbacks = ['release']
-        }
         debug {
             shrinkResources false
             minifyEnabled false
             applicationIdSuffix ".debug"
             manifestPlaceholders.isRaptorEnabled = "true"
         }
+        // "releaseRaptor" is only used for performance testing, and isn't a real "release" type
+        releaseRaptor releaseTemplate >> { // the ">>" concatenates the releaseRaptor-specific options with the template
+            manifestPlaceholders.isRaptorEnabled = "true"
+        }
+        nightly releaseTemplate
+        beta releaseTemplate
```

Comment:
Why do we have a different beta buildType? Are we planning to have different store entries like Fennec? Or are we planning to release the release version to beta before release like Focus?

We can push the same binary to beta and then to release, but we'll have to decide how to indicate to users if they're on the beta channel. Google doesn't tell us what track the user is on.

Reply:
Good question, this was confusing with `release` still being in there. Now, we have just `nightly` and `beta` build types.

 How we handle releasing (fennec/focus) is still fluctuating at the moment, so I've got no good answers there :)